tests/chunkAssessment.test.js

```diff
@@ -1,6 +1,7 @@
 const util = require('util');
 const assert = require('chai').assert;
 const { step } = require('mocha-steps');
+const tmp = require('tmp-promise');
```

Comment:
I'm not 100% sure why this was necessary, but without this, the tests failed because they couldn't create the path `/chunks`. This may have been unique to me because I develop outside of Docker. At any rate, the new setup mirrors what happens in `tests/chunks`.

Reply:
The new setup certainly seems cleaner in any case.